.changeset/rich-shirts-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@qwik.dev/core': patch
```

Comment:
@Varixo @wmertens  technically this is not a patch, it's a minor (we're adding an API surface) no?

Reply:
no - it's for v2 and it's repairing missing API surface